### DISCUSSION TITLE
mode=CAR on API expected to return an empty result?

### DISCUSSION THREAD

**Discussion by 5balls** [motis-project/motis#591] [Q&A] [answered]
This API request works great:

https://europe.motis-project.de/api/v1/plan?fromPlace=50.94262,6.95589&toPlace=51.33964,12.37185&mode=WALK,TRANSIT&maxHours=12

This one however doesn't:

https://europe.motis-project.de/api/v1/plan?fromPlace=50.94262,6.95589&toPlace=51.33964,12.37185&mode=CAR&maxHours=12

neither does this:

https://europe.motis-project.de/api/v1/plan?fromPlace=50.94262,6.95589&toPlace=51.33964,12.37185&mode=WALK,CAR&maxHours=12

Is this to be expected and a user error? Does motis need another routing engine for car only routing?

**Comment by 5balls** [comment]
For car mode I get:
```

{
  "requestParameters": {},
  "debugOutput": {},
  "date": 0,
  "from": {
    "name": "",
    "lat": 50.94262,
    "lon": 6.95589,
    "level": 0,
    "vertexType": "NORMAL"
  },
  "to": {
    "name": "",
    "lat": 51.33964,
    "lon": 12.37185,
    "level": 0,
    "vertexType": "NORMAL"
  },
  "itineraries": [],
  "previousPageCursor": "EARLIER|1729336080",
  "nextPageCursor": "LATER|1729336080"
}
```

**Comment by felixguendling** [comment, marked as answer]
This is not yet implemented. It will be until end of beta phase.

If you really need car routing, there currently is only the `one-to-many` endpoint which only gives you the duration:
https://europe.motis-project.de/api/v1/one-to-many?one=49.85641729052107;8.63964747373378&many=49.881513079267535;8.65855340640723&mode=WALK&max=7200&maxMatchingDistance=200&arriveBy=false

The routing API will support `CAR`, `BIKE` and `FOOT` routing in the next few weeks.

**Comment by 5balls** [comment]
Ah ok. It is great to hear that this will be implemented! I will use graphhopper until then but hope to get rid of it once motis 2.0.0 is finally out.
